packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -465,6 +465,28 @@ You can disable specific rows by providing an array of keys to TableView via the
 <PokemonTable selectionMode="multiple" disabledKeys={[3]} />
 ```
 
+### Highlight selection
+
+By default, TableView uses the checkbox selection style, which includes a column of checkboxes in each row for selection. When the `selectionStyle` prop is set to `"highlight`", the checkboxes are hidden, and the selected rows are displayed with a highlighted background instead.
+
+In addition to changing the appearance, the selection behavior also changes depending on the `selectionStyle` prop. In the default checkbox selection style, clicking, tapping, or pressing the <kbd>Space</kbd> or <kbd>Enter</kbd> keys toggles selection for the focused row. Using the arrow keys moves focus but does not change selection.
+
+In the highlight selection style, clicking a row with the mouse _replaces_ the selection with only that row. Using the arrow keys moves both focus and selection. To select multiple rows, modifier keys such as <kbd>Ctrl</kbd>, <kbd>Cmd</kbd>, and <kbd>Shift</kbd> can be used. On touch screen devices, selection always behaves as toggle since modifier keys may not be available. This behavior emulates native platforms such as macOS and Windows.
```

Comment:
Do we wanna mention using option (mac)/ctrl (windows) + arrow to move focus w/o changing selection?Perhaps that is expected knowledge for windows users since it emulates File explorer native behavior but might be unknown for Mac users since option + arrow doesn't work in finder